suclass/cms/templates/body-post.html

```diff
@@ -1 +1,14 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+## We have to wait to append this script to the page,
+## otherwise there's a chance the zendesk widget does
+## not show up on the page. Most likely, due to a race
+## condition where other elements of the page must load first.
+<script>
+  $(document).ready(function(){
+    setTimeout(function (){
+      var script = document.createElement('script');
+      script.id = 'ze-snippet';
+      script.src = 'https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85';
+      $('body').append(script);
+    }, 500); // Smallest value that guaranteed zendesk widget to show consistently
```

Comment:
@caesar2164 We were able to get this down smaller.
The explanation is slightly wonky; give a holler if you want a rundown F2F.